test/test_jit_fuser.py

```diff
@@ -60,6 +62,21 @@ def func(x):
     def test_abs_cpu(self):
         self._test_fused_abs()
 
+    @unittest.skipIf(IS_SANDCASTLE, "NYI: fuser CPU support for Sandcastle")
+    @enable_cpu_fuser
+    def test_abs_cpu_unicode_temp_dir(self):
+        if IS_FILESYSTEM_UTF8_ENCODING:
+            with TemporaryDirectoryName(suffix='中文') as dname:
+                shell_env = os.environ.copy()
+                shell_env['TMP'] = dname
```

Comment:
Note that `TMP` is searched before `TEMP`. On my machine there's already `TMP`. So this has to overwrite `TMP`. Using `TEMP` won't work.

Reply:
Yes, this is consistent with the docs. https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-gettemppatha